src/js/themes/colors.js

```diff
@@ -37,10 +37,13 @@ export const colors = {
     light: '#000000',
   },
   'text-weak': {
-    dark: '#606B7D',
-    light: '#BBBBBB',
+    dark: '#8c98aa',
+    light: '#757575',
+  },
+  'text-xweak': {
+    dark: '#606b7d',
```

Comment:
small change, but for consistency with the rest of theme can we capitalize the letters in these HEX values?